posts/2023-10-20-open-liberty-with-cargo-tracker-ee10.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Running Eclipse Cargo Tracker, a Jakarta EE exemplar, with Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/gkwan-ibm
+author_github: https://github.com/gkwan-ibm
+seo-title: Run Eclipse Cargo Tracker, an exemplar Jakarta EE 10 application, with Open Liberty - OpenLiberty.io
+seo-description: Experience Jakarta EE 10 and Open Liberty in an industrial environment by running Eclipse Cargo Tracker, an open-source application built to simulate real-world implementations of Jakarta EE 10.
+blog_description: "Experience Jakarta EE and Open Liberty in an industrial environment by running Eclipse Cargo Tracker, an open-source application built to simulate real-world implementations of Jakarta EE."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running Eclipse Cargo Tracker, a Jakarta EE exemplar, with Liberty
+Gilbert Kwan <https://github.com/gkwan-ibm>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+In September, the comprehensive https://eclipse-ee4j.github.io/cargotracker/[Eclipse Cargo Tracker] exemplar application was updated with several key milestones, including targeting Jakarta EE 10 and support for key runtimes including Open Liberty as well as Payara and Eclipse Glassfish. This means that you can now learn all about Jakarta EE in an industrial standard by running Cargo Tracker with Open Liberty, all on your local machine.
+
+
+== What is the Cargo Tracker application?
+
+As one of the top Java framework choices for enterprise-level and cloud-native Java applications, Jakarta EE is at the center of projects all over the globe. The Cargo Tracker open-source application was designed to replicate a complex enterprise application by using core Jakarta EE features with professional development standards.
```

Comment:
I would remove level and and, so instead have "for cloud-native enterprise Java applications"

Reply:
Switch ordering in next sentence to: "The open-source Cargo Tracker application"